css/index.css

```diff
@@ -1,3 +1,328 @@
+                            margin: 2%;
+                        }
+                        
+                        .prod-main {
+                            border-top: 2px solid black;
+                            border-bottom: 2px solid black;
+                            padding-bottom: 3%;
+                            display: flex;
+                            flex-flow: wrap;
+                            justify-content: space-around;
+                        }
+                        
+                        .prod-offer {
+                            margin-top: 3%;
+                            background-color: #BFE600;
+                            padding: 3%;
+                            width: 45%;
+                            border-radius: 10px;
+                        }
+                        
+                        .prod-offer p {
+                            margin: 5%;
+                        }
+                        
+                        .prod-lower {
+                            display: flex;
+                            justify-content: space-around;
+                        }
+                        
+                        .prod-bottom {
+                            padding: 3%;
+                            display: flex;
+                            flex-direction: column;
+                            width: 45%;
+                        }
+                        
+                        .prod-bottom ul {
+                            list-style-type: square;
+                            margin-top: 2%;
+                        }
+                        
+                        .prod-info-img {
+                            margin-top: 3%;
+                            margin-left: 10%;
+                        }
+                        /*  -------------------product page---------^^^^^^^^^^^^^^^^^^^----*/
+                        /*---------------------media queries-------vvvvvvvvvvvvvvvvvv-------*/
+                        
+                        @media(max-width: 800px) {
+                            .head {
+                                width: 100%;
+                                margin: 3%;
+                                padding: 5%;
+                                display: flex;
+                                flex-direction: column-reverse;
+                                justify-content: space-evenly;
+                                align-items: center;
+                            }
+                            .head-main img {
+                                display: none;
+                            }
+                            .svc-offer {
+                                width: 100%;
+                            }
+                            .services-info-img {
+                                display: none;
+                            }
+                        }
+                        
+                        @media(max-width: 500px) {
```

Comment:
Since you set margin and padding in @media-800px, you'd need to remove that when you set @media-500px's margin and padding:

padding: 0;
margin: 0;

Then trial and error with the rest to get the nav links to spread across the mobile page.